modules/apps/shared-dependencies/shared-dependencies-groovy-all/bnd.bnd

```diff
@@ -0,0 +1,71 @@
+Bundle-Name: Liferay Shared Dependencies Groovy All
+Bundle-SymbolicName: com.liferay.shared.dependencies.groovy.all
+Bundle-Version: 1.0.0
+Import-Package:\
+	!com.thoughtworks.qdox.*,\
+	!com.thoughtworks.xstream.*,\
+	\
+	!groovyjarjarasm.asm.tree.analysis.*,\
+	\
+	!jline.*,\
+	\
+	!junit.*,\
+	\
+	!org.apache.bsf.*,\
+	!org.apache.commons.cli.*,\
+	!org.apache.ivy.*,\
+	!org.apache.tools.*,\
+	\
+	!org.fusesource.jansi.*,\
+	\
```

Comment:
@marianoalvarosaiz, there are two issues at here.
1.You add ` !org.apache.tools.*,\` at here, but you didn't remove this from `com.liferay.commerce.report.impl` and `com.liferay.portal.reports.engine.console.jasper`

2. It missed `!javax.swing.*,\`

Reply:
I've explained now in the commit message why it is needed (due to other included libraries).